### DISCUSSION TITLE
Iliadbox

### DISCUSSION THREAD

**Discussion by fragrusti** [localsend/localsend#1339] [L-Italiano]
Buonasera,

ho una fibra iliad con iliadbox r1 aggiornata all'ultimo firmware con bande wifi separate e con un NAS in rete completamente funzionante (samba, FTP, https etc). Per quanto io faccia Localsend tra MacOS, iOS, Android non trova alcun dispositivo e non funziona nemmeno il link diretto e mi ritorna un generico Errore.

Credo sia colpa dell'iliadbox ma non capisco come ovviare, qualcuno può aiutarmi?

Grazie in anticipo.
Ciao,
f

**Comment by LuxyRepo** [comment]
cosa intendi con rete wifi separata? hai un sistema mesh?
